src/tree/registries/registryContextValues.ts

```diff
@@ -0,0 +1,63 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
What was the design decision to create this file to parse all of the contextValues instead of letting the `treeItemBase` classes contain the values?  I was wondering since it's a pretty different design pattern from our other repos.

Reply:
I wanted to create a common pattern for context values so that people wouldn't have to edit package.json when adding new registry providers. The [GitLab PR](https://github.com/microsoft/vscode-docker/pull/1098) shows that. I found that the previous pattern wasn't flexible enough to support that.